cves/CVE-2017-9787.yml

```diff
@@ -139,8 +151,8 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: ognl
```

Comment:
What was your reasoning behind placing this in the "ognl" subsystem?

Reply:
Added reasoning